### PR TITLE
Feature/admin view only

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/adjudication/pages/enrolments/enrolments.component.ts [hunk 1]
```diff
@@ -73,10 +75,6 @@ export class EnrolmentsComponent implements OnInit {
     return (currentStatusCode === EnrolmentStatus.UNDER_REVIEW);
   }
 
-  public isSuperAdmin(): boolean {
```
Comment (AnaisHebert) [inline prime-angular-frontend/src/app/modules/adjudication/pages/enrolments/enrolments.component.ts]: ERROR in src/app/modules/adjudication/pages/enrolments/enrolments.component.html(178,15): Property 'isSuperAdmin' does not exist on 

this is till used on the template i believe

Comment (hogmann) [reply prime-angular-frontend/src/app/modules/adjudication/pages/enrolments/enrolments.component.ts]: Whoops, this is why I'm supposed to build prod after changes

File: prime-dotnet-webapi/AuthenticationSetup.cs [hunk 2]
```diff
@@ -90,8 +90,10 @@ IHostEnvironment environment
             services.AddAuthorization(options =>
             {
                 options.AddPolicy(PrimeConstants.USER_POLICY, policy => policy.Requirements.Add(new PrimeUserRequirement()));
+                //options.AddPolicy(PrimeConstants.ACCESS_POLICY, policy => policy.RequireRole().a)
```
Comment (AnaisHebert) [inline prime-dotnet-webapi/AuthenticationSetup.cs]: Should this commented line be here?

Comment (hogmann) [reply prime-dotnet-webapi/AuthenticationSetup.cs]: It should not be there.

**Pull request merged**